phpcs.xml.dist

```diff
@@ -7,10 +7,305 @@
 
   <rule ref="Drupal">
```

Comment:
as there is no rule anymore listeed here, let's fully remove this tag completely